server/src/main/java/org/apache/iotdb/db/engine/compaction/TsFileIdentifier.java

```diff
@@ -43,11 +43,11 @@
   public static final int LOGICAL_SG_OFFSET_IN_PATH = 4;
   public static final int SEQUENCE_OFFSET_IN_PATH = 5;
 
-  public static final int LOGICAL_SG_OFFSET_IN_LOG = 0;
-  public static final int VIRTUAL_SG_OFFSET_IN_LOG = 1;
-  public static final int TIME_PARTITION_OFFSET_IN_LOG = 2;
-  public static final int FILE_NAME_OFFSET_IN_LOG = 3;
-  public static final int SEQUENCE_OFFSET_IN_LOG = 4;
+  public static final int SEQUENCE_OFFSET_IN_LOG = 0;
```

Comment:
After these offsets are changed, the sample in the comments of method `getFileIdentifierFromInfoString` is not correct

Reply:
resolved